tests/ui/transmutability/maybeuninit.rs

```diff
@@ -22,12 +22,12 @@ fn validity() {
 }
 
 fn padding() {
-    #[repr(align(8))]
-    struct Align8;
+    #[repr(align(4))]
```

Comment:
This temporary downsizing is tracked here: https://github.com/rust-lang/project-safe-transmute/issues/25